Makefile

```diff
@@ -12,4 +12,5 @@ lint:
 	pipenv check ./secure_message ./tests
 
 test: lint
-	pipenv run python run_tests.py
+	pipenv run behave --format progress
+	export APP_SETTINGS=TestConfig && pipenv run pytest && unset APP_SETTINGS
```

Comment:
I don't think exporting this here will have any effect on the environment in which pipenv is executing the pytest script from. You'll need to use a .env for that or run the export and unset inside the pipenv shell